### DISCUSSION TITLE
How to always render 4 images in 1 run?

### DISCUSSION THREAD

**Discussion by shagratt** [Acly/krita-ai-diffusion#832] [Q&A] [answered]
Sometimes when I press the "generate" button i get 4 images created after procesing. I cant understand why sometimes it only create 1 or 2 instead of 4. Is there a hidden setting for this? Depends on something? Can anyone explain me this please.
(Clarification: what I'm asking is not the same as putting 4 images in queue setting batch to 4, but creating 4 images per batch)

**Comment by Acly** [comment, marked as answer]
It's controlled by Batch Size in Performance settings tab. Measured in multiples of 512x512 images, so it depends on your resolution, but keeps VRAM requirements constant.

**Comment by shagratt** [comment]
Ahhh I got it on automatic thats why sometimes I get less images. Thanks Acly!  😀​👍​
